devices/fields/dac/g4.yaml

```diff
@@ -0,0 +1,51 @@
+_include:
+  - v6.yaml
+
+SWTRGR:
+  SWTRIGB[12]:
+    NoTrigger: [0, No trigger]
+    Trigger: [1, Trigger for sawtooth increment]
+
+STR[12]:
+  STRSTDATA[12]: [0, 0xFFF]
+  STDIR[12]:
+    Decrement: [0, Decrement]
+    Increment: [1, Increment]
+  STINCDATA[12]: [0, 0xFFFF]
+
+CR,STMODR:
+  TSEL?,STRSTTRIGSEL?:
+    Swtrig: [0, Software trigger]
+    Tim8Trgo: [1, Timer 8 TRGO event]
```

Comment:
From what I can tell from Table 188 in [RM0440](https://www.st.com/resource/en/reference_manual/rm0440-stm32g4-series-advanced-armbased-32bit-mcus-stmicroelectronics.pdf), DAC3 has Tim1Trgo here instead of Tim8Trgo. Same thing for STINCTRIGSEL

Reply:
Renamed to `Tim1or8Trgo`. In other case it is needed to copy DAC2-4 instead of derive.

Reply:
Makes sense 👍